acceptance-tests/build.gradle

```diff
@@ -45,6 +46,9 @@ task acceptanceTest(type: Test) {
   description = 'Runs Eth2Signer acceptance tests.'
   group = 'verification'
 
+  setSystemProperties(test.getSystemProperties())
```

Comment:
nit: what's this required for? Does it effectively copy a test's properties to the child process?